src/Kaleidoscope-LEDEffect-Rainbow.h

```diff
@@ -19,26 +19,30 @@ class LEDRainbowEffect : public LEDMode {
   uint16_t rainbow_ticks = 10;  //  delays between update
 
   byte rainbow_saturation = 255;
-  byte rainbow_value = 50;
+  byte rainbow_value = 151;
 };
 
 
 class LEDRainbowWaveEffect : public LEDMode {
  public:
-  LEDRainbowWaveEffect(void) {}
+  LEDRainbowWaveEffect(void) {} //  Empty constructor with default values
+  LEDRainbowWaveEffect(byte);
+  LEDRainbowWaveEffect(byte, uint16_t);
+  LEDRainbowWaveEffect(byte, uint16_t, uint8_t);
 
   void brightness(byte);
+  void delay(uint16_t);
+  void precision(uint8_t);
   void update(void) final;
 
  private:
   uint16_t rainbow_hue = 0;  //  stores 0 to 614
-
   uint8_t rainbow_wave_steps = 1;  //  number of hues we skip in a 360 range per update
-  uint16_t rainbow_current_ticks = 0;
+  uint16_t rainbow_current_ticks = 0; //  the current tick in the cycle
   uint16_t rainbow_wave_ticks = 10;  //  delays between update
 
   byte rainbow_saturation = 255;
-  byte rainbow_value = 50;
+  byte rainbow_value = 151;
```

Comment:
Why the change from 50 to 151?

(I have no recollection where and how this value is used, but the change does not seem to be explained neither in the commit messages, nor the PR)